### PR TITLE
GHSA Sync: Added 6 brand new jquery-related advisories

### PR DESCRIPTION
**Pull request by jasnow**
GHSA Sync: Added 6 brand new jquery-related advisories.

### CODE REVIEW

File: gems/jquery-rails/CVE-2020-11022.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  "### Impact\nPassing HTML from untrusted sources - even after sanitizing
+  it - to one of jQuery's DOM manipulation methods (i.e. `.html()`, `.append()`, and
+  others) may execute untrusted code.\n\n### Patches\nThis problem is patched in jQuery
+  3.5.0.\n\n### Workarounds\nTo workaround the issue without upgrading, adding the
+  following to your code:\n\n```js\njQuery.htmlPrefilter = function( html ) {\n\treturn
+  html;\n};\n```\n\nYou need to use at least jQuery 1.12/2.2 or newer to be able to
+  apply this workaround.\n\n### References\nhttps://blog.jquery.com/2020/04/10/jquery-3-5-0-released/\nhttps://jquery.com/upgrade-guide/3.5/\n\n###
+  For more information\nIf you have any questions or comments about this advisory,
+  search for a relevant issue in [the jQuery repo](https://github.com/jquery/jquery/issues).
+  If you don't find an answer, open a new issue."
```
Comment (postmodern) [inline gems/jquery-rails/CVE-2020-11022.yml:18]: This appears to be a double-escaped String? Is this a bug in the github sync script?

File: gems/jquery-ui-rails/CVE-2021-41183.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+gem: jquery-ui-rails
+cve: 2021-41183
+ghsa: j7qv-pgf6-hvh4
+url: https://github.com/jquery/jquery-ui/security/advisories/GHSA-j7qv-pgf6-hvh4
+title: XSS in `*Text` options of the Datepicker widget in jquery-ui
+date: 2021-10-26
+description: |
+  "### Impact\nAccepting the value of various `*Text` options of the Datepicker
```
Comment (postmodern) [inline gems/jquery-ui-rails/CVE-2021-41183.yml:9]: Also double escaped.

File: gems/jquery-ui-rails/CVE-2021-41184.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  util from untrusted sources may execute untrusted code. For example, invoking the
+  following code:\n```js\n$( \"#element\" ).position( {\n\tmy: \"left top\",\n\tat:
+  \"right bottom\",\n\tof: \"<img onerror='doEvilThing()' src='/404' />\",\n\tcollision:
+  \"none\"\n} );\n```\nwill call the `doEvilThing()` function.\n\n### Patches\nThe
+  issue is fixed in jQuery UI 1.13.0. Any string value passed to the `of` option is
+  now treated as a CSS selector.\n\n### Workarounds\nA workaround is to not accept
+  the value of the `of` option from untrusted sources.\n\n### For more information\nIf
+  you have any questions or comments about this advisory, search for a relevant issue
+  in [the jQuery UI repo](https://github.com/jquery/jquery-ui/issues). If you don't
+  find an answer, open a new issue."
```
Comment (postmodern) [inline gems/jquery-ui-rails/CVE-2021-41184.yml:19]: Double escaped.

**Review by postmodern — CHANGES_REQUESTED** [review]
Appears that some of the `description` got double escaped. I guess this is due to some edge-case in the github sync script?

They can be un-escaped using `puts YAML.load(<paste-double-quoted-string-here>)` or just `puts <paste-double-quoted-string-here>`.

File: gems/jquery-ui-rails/CVE-2021-41183.yml (hunk 2 above)
Comment (jasnow) [reply gems/jquery-ui-rails/CVE-2021-41183.yml]: Corrected

File: gems/jquery-ui-rails/CVE-2021-41184.yml (hunk 3 above)
Comment (jasnow) [reply gems/jquery-ui-rails/CVE-2021-41184.yml]: Corrected

File: gems/jquery-rails/CVE-2020-11022.yml (hunk 1 above)
Comment (jasnow) [reply gems/jquery-rails/CVE-2020-11022.yml:37]: Corrected

**Review by postmodern — APPROVED** [review]

**Pull request merged**